Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -91,16 +91,51 @@ YUI.add('ez-relation-editview', function (Y) {
          * @return Object
          */
         _variables: function () {
-            var dest = this.get('destinationContent');
+            var destinationContent = this.get('destinationContent'),
+                destinationContentJSON = null,
+                isLoaded = false;
+
+            if (destinationContent !== null) {
+                isLoaded = true;
+
+                if (this._isNewRelation(destinationContent) || !this._isTrashed(destinationContent)) {
+                    destinationContentJSON = destinationContent.toJSON();
+                }
+            }
 
             return {
-                destinationContent: dest ? dest.toJSON() : null,
+                destinationContent: destinationContentJSON,
                 loadingError: this.get('loadingError'),
                 isEmpty: this._isFieldEmpty(),
+                isLoaded: isLoaded,
```

Comment:
I don't know what version of JS we use, but in ES6 you can just write:
```
return {
//...
isLoaded,
//...
```
Not shure if it is viable here.

Reply:
@tischsoic in PlatformUI we have to follow ES5 standards as it uses an outdated version of PhantomJS for tests. If we start using ES6 features even `bind()` then the tests will be broken due to outdated tools. There is no willing to update these tools.